study.md

```diff
@@ -22,7 +22,8 @@ material. Instead, digest what you've read and repeat it in your own voice.
 In your own words, define ORM and explain why using an ORM is valuable.
 
 ```md
-<!-- your response here -->
+ORM is a way to convert data between object and table. Using ORM to store and retrieve data  from a database
```

Comment:
Object relational-mapping (ORM) is a method of representing arbitrary data as objects in a programming language.  They are valuable because they provide an easy-to-use API for database-related tasks.